microsoft-edge/webview2/webview2-api-reference.md

```diff
@@ -19,11 +19,11 @@ WebView2 is available for the following frameworks (or platforms) and programmin
    * [WPF](/dotnet/api/microsoft.web.webview2.wpf)
    * [Windows Forms](/dotnet/api/microsoft.web.webview2.winforms)
 
-* WinRT for WinUI 2 (UWP)
-   * [Core/C#](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/index)
-   * [COM Interop/C++](/microsoft-edge/webview2/reference/winrt/interop/index)
+* WinRT for WinUI 2 (UWP) 
+   * [Core](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/index)
+   * [COM Interop/C++](/microsoft-edge/webview2/reference/winrt/interop/index) 
 
-* WinRT for WinUI 3 (Windows App SDK)
+* WinRT for WinUI 3 (Windows App SDK) 
    * [Microsoft.UI.Xaml.Controls.WebView2 Class](/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.webview2)
       * [Spec: WebView2 Xaml control](https://github.com/microsoft/microsoft-ui-xaml-specs/blob/master/active/WebView2/WebView2_spec.md)<!-- changing master to main doesn't work 5/19/2022 -->
```

Comment:
[This API spec ](https://github.com/microsoft/microsoft-ui-xaml-specs/blob/master/active/WebView2/WebView2_spec.md) is out of date and should be made unnecessary by the corresponding winui3 webview2 ref docs